.github/workflows/run_tests.yml

```diff
@@ -8,7 +8,7 @@ on:
     types: [opened, synchronize, reopened]
 
 env:
-  POETRY_VERSION: "1.6.1"
+  POETRY_VERSION: "1.7.1"
```

Comment:
In the poetry lock file, it looks like it's getting compiled with `1.7.1` - I'd recommend updating the version in CI to reflect what developers are using locally.